ui/src/authors/components/OrcidPushSettingMessage.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ExternalLink from '../../common/components/ExternalLink';
+import OrcidProfileLink from '../../common/components/OrcidProfileLink';
+
+function OrcidPushSettingMessage({ orcid, enabled, authorBAI }) {
```

Comment:
just a suggestion since it's mostly hardcoded text, we can separate this if blocks

Reply:
I was thinking so, but then thought it's easy to read and understand what's going on, you don't think so?